.github/actions/setup-opa/action.yaml

```diff
@@ -0,0 +1,13 @@
+name: Setup OPA CLI
+description: Setup OPA CLI
+runs:
+  using: composite
```

Comment:
Nice! besides refactoring into a separate YAML, what other advantages does it have?

Reply:
Added the retry flag to repeat the operation after a failure and changed the downloading of the OPA from the site to GitHub (I think GitHub is a more reliable source)